AccessibilityInsightsForAndroidService/.gitignore

```diff
@@ -1,15 +1,16 @@
-*.iml
+.cxx
+.DS_Store
+.externalNativeBuild
 .gradle
-/local.properties
+*.iml
+/.idea/assetWizardSettings.xml
 /.idea/caches
+/.idea/jarRepositories.xml
```

Comment:
`/.idea/jarRepositories.xml` is the only addition/change here ,the rest of the changes in this file are sorting the lines.

`jarRepositories` is a new file that Android Studio learned to auto-generate with the updated gradle build tool version. See [this StackOverflow answer](https://stackoverflow.com/a/63544057) for an explanation of why we're .gitignoring it rather than checking it in.